### PR TITLE
Allow material-ui 0.17.x in peerDependencies and add yarn.lock

### PR DESCRIPTION
**Pull request by hhaidar**
We're using material-ui v0.17 and this component works well with it!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "react-tap-event-plugin": "^2.0.0"
   },
   "peerDependencies": {
-    "material-ui": "~0.16.7",
+    "material-ui": ">=0.16 <=0.17",
```
Comment (sibartlett) [inline package.json]: Why not do the following? `^0.16`

I don't see a reason to limit the peer dependency to a fixed range, unless someone reports an issue.

Comment (hhaidar) [inline package.json]: Yea I think we should do that too then.

Comment (leMaik) [inline package.json]: @hhaidar Feel free to change it. :+1:

**Comment by sibartlett** [comment]
This looks good to merge :)

**Comment by leMaik** [comment]
Indeed. :+1:

**Comment by leMaik** [comment]
Published as [v1.1.2](https://github.com/TeamWertarbyte/material-ui-password-field/releases/tag/v1.1.2). :fire_engine:

**Comment by hhaidar** [comment]
Thanks, much appreciated @leMaik!

**Pull request merged**